pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -191,6 +220,15 @@ func (w *Watcher) init() error {
 // Goroutines started here will be waited for in Stop() before cleaning up.
 // Ignore all errors except root dir not being walkable
 func (w *Watcher) traversePluginDir(dir string) error {
```

Comment:
this is called at times other than Start()... checking for isPluginDir() doesn't ensure that scanned synthetic events from new directories we encounter are processed before observed events for those new directories

Reply:
Done